Sources/NIO/SelectorEpoll.swift

```diff
@@ -0,0 +1,290 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the SwiftNIO open source project
+//
+// Copyright (c) 2021 Apple Inc. and the SwiftNIO project authors
```

Comment:
given that this code has mostly been moved, I think this needs to be 2017-2021

Reply:
Fixed in e19e685.